package.json

```diff
@@ -12,13 +12,14 @@
     "@types/react-router-hash-link": "^2.4.1",
     "@vanilla-extract/css": "^1.2.3",
     "@vanilla-extract/css-utils": "^0.1.1",
+    "babel-loader": "^8.2.3",
```

Comment:
We shouldn't rely on a transitive dependency for https://github.com/seek-oss/scoobie/blob/3d6a16341044324324b278f7e711c54d7c545d28/webpack/webpackPlugin.js#L19